test/common.py

```diff
@@ -2433,23 +2472,32 @@ def browser_open(cls, url):
   @classmethod
   def setUpClass(cls):
     super().setUpClass()
+    # In parallel mode this ID contains which pool worker this process is.
+    cls.WORKER_ID = get_worker_id()
+    cls.PORT = 8888 + cls.WORKER_ID
+    cls.SERVER_URL = f'http://localhost:{cls.PORT}'
+    cls.HARNESS_URL = f'{cls.SERVER_URL}/run_harness'
```

Comment:
Can this stuff be done at the class level instead?   

Also, we probably should be using ALL_CAPS for these.. although maybe we don't need to change that in this PR

Reply:
These need to be assigned later than the class level otherwise they all end up with the same value of the initial forked process.

Reply:
Why is that?  Does `get_worker_id()` not work at load time?